sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -436,8 +436,9 @@ def finish_bundle(self):
       bundles = [bundle]
     else:
       bundles = []
-    input_pvalue = self._applied_ptransform.inputs
-    if not input_pvalue:
+    if self._applied_ptransform.inputs:
```

Comment:
Thanks, this is correct.  The currently checked-in version has a bug where we fail to extract the correct input PValue, which is fixed by David's change here.